examples/test-with.py

```diff
@@ -1,80 +1,83 @@
-from ritz import ritz, notifier, parse_tcl_config
-from pprint import pprint
-from os.path import expanduser
-from time import sleep
-import re
 import argparse
+import re
 import sys
+from os.path import expanduser
+from pprint import pprint
+from time import sleep
+
```

Comment:
isort has a flag for whether it should sort on the whole line (including `import` and `from`)  or skip the first word. I prefer the latter because you get:

```
import os
from os.path import expanduser
import pprint
```

instead of 

```
import os
import pprint
from os.path import expanduser
```

We should probably settle on what we want before we lock things in.